vpr/src/route/route_timing.cpp

```diff
@@ -1608,7 +1608,7 @@ static bool early_exit_heuristic(const t_router_opts& router_opts, const Wirelen
     return false;
 }
 
-static OveruseInfo calculate_overuse_info() {
+static OveruseInfo calculate_overuse_info(std::vector<ClusterNetId>& rerouted_nets) {
```

Comment:
Minor, but we should probably be passing this as `const std::vector<ClusterNetId>&` since it isn't modified in this function